app/models/workflow.js

```diff
@@ -9,17 +9,29 @@ const {
 
 export default Model.extend({
     title: attr('string'),
+    parameters: hasMany('parameter', {
+        inverse: 'workflow'
+    }),
     description: attr('string'),
+    sortedSections: Ember.computed('sections.@each.index', function() {
```

Comment:
in the hasmany, order is not guaranteed for the related records. The recors thus have an index, and are sorted. Anywhere we nee them in order, like rendering in the template, soretedSections is used.